app/src/pages/inside/historyPage/historyView/historyTable/historyItemsGrid/historyItem/historyItem.scss

```diff
@@ -0,0 +1,98 @@
+.history-item-wrapper {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+  min-width: 62px;
+  min-height: 50px;
+  padding: 4px;
+  font-size: 12px;
+  text-align: center;
+  font-family: $FONT-REGULAR;
+  line-height: 1;
+}
+
+.history-item-status-FAILED {
+  background-color: lighten($COLOR--failed, 10%);
```

Comment:
Do not use color functions if you are not sure that it's required. If you see some new color, which is not represented in color palette, it's a reason to talk with UX Designer. The same, if you see the difference between BB version and Zeplin.
Wrong colors, see zeplin to get correct.
https://app.zeplin.io/project/56f5171994e9f9943d6098de/screen/58405b7724bf034809b82854